src/auth/auth.service.ts

```diff
@@ -19,6 +28,7 @@ export class AuthService {
         const payload = { username: user.email, sub: { employeeId: user.employee.employeeId, companyId: user.employee.companyId, role: user.employee.role } };
         return {
             access_token: this.jwtService.sign(payload),
+            refreshToken: await this.generateRefreshToken(user.email),
```

Comment:
similar to previous comments, this token should be signed using the jwt service - this will allow for unsigning/decrypting the the token like we do for the jwt without storing in the db